content/_includes/embed.macros.njk

```diff
@@ -256,7 +256,7 @@ params:
       page,
       'data.events',
       {'venue': venue} if venue else none
-    ) if page else
+    ) | first if page else
```

Comment:
Seems like there was a bug in this macro when `page` was set -- `event` was a list, where we just want the first one (equivalent to `findData`).